src/client/app/actions/uiActions.js

```diff
@@ -1,10 +1,12 @@
 import { TOGGLE_PLAYLIST_DRAWER } from '../constants';
 
-const togglePlaylistDrawer = (val) => ({
+const togglePlaylistDrawer = val => ({
     type: TOGGLE_PLAYLIST_DRAWER,
     data: {
         playlistDrawer: val
     }
 });
 
-export { togglePlaylistDrawer };
+const dummyAction = lint => console.log(lint);
```

Comment:
@jcuratolo the linter wants you to use the `default` export if there's only one thing being exported, hence dummy lint function. There's a 100% chance there will be more things to export